example/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ add_subdirectory(28_grouped_gemm_bias_e_permute)
 add_subdirectory(29_batched_gemm_bias_e_permute)
 add_subdirectory(30_grouped_convnd_fwd_bias_relu_add)
 add_subdirectory(31_batched_gemm_gemm)
-add_subdirectory(32_batched_gemm_softmax_gemm)
+add_subdirectory(32_batched_gemm_scale_softmax_gemm)
```

Comment:
I also have the impression that CK doesn't advertise the elementwise op capability in the function name because it implicitly supports all elementwise ops in the pipe. So I feel "scale" is redundant as it's already captured by Acc0 elementwise op

Reply:
it does indicate that in the example file name...

it doesn't indicate in the DeviceOp name though

Reply:
Makes sense. Thanks for the explanation!